sites/www.thunderbird.net/includes/canonical-url.html

```diff
@@ -0,0 +1,45 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<link rel="canonical" href="{{ settings.CANONICAL_URL + '/' + LANG + canonical_path }}">
+{% if is_homepage %}<link rel="alternate" hreflang="x-default" href="{{ settings.CANONICAL_URL }}{{ canonical_path }}">{% endif %}
+{% if translations and canonical_path -%}
+  {%- for code, label in translations|dictsort -%}
+    {%- if code == 'en-US' -%}
```

Comment:
Could we update `settings.CANONICAL_LOCALES` and use that info (flipped ofc) for these special cases? This is fine though.

Reply:
yeah good point